kubernetes/ama-logs.yaml

```diff
@@ -526,6 +531,11 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.hostIP
+            - name: CONTAINER_MEMORY_LIMIT
```

Comment:
Include the units in the environment variable name. and also, check the units are constant regardless of the units specified on the resource limit.

Reply:
done